runtime/onert/core/src/compiler/OperationValidator.cc

```diff
@@ -107,7 +107,7 @@ void OperationValidator::visit(const ir::operation::ElementwiseUnary &node)
   if (node.param().op_type == ir::operation::ElementwiseUnary::Type::DEQUANTIZE)
   {
     OP_REQUIRES((_ctx.at(input_index).typeInfo().type() == ir::DataType::QUANT_UINT8_ASYMM) ||
-                (_ctx.at(input_index).typeInfo().type() == ir::DataType::QUANT_INT8_SYMM));
+                (_ctx.at(input_index).typeInfo().type() == ir::DataType::QUANT_INT8_ASYMM));
```

Comment:
`Dequantize` TCs get input as `TENSOR_QUANT8_SYMM` type. It seems that TC fails at here.

Reply:
@periannath I've updated. PTAL